src/coreclr/src/jit/optcse.cpp

```diff
@@ -2461,6 +2461,28 @@ class CSE_Heuristic
                     extra_yes_cost *= 2; // full cost if we are being Conservative
                 }
             }
+
+#ifdef FEATURE_SIMD
+            // SIMD types will cause a SIMD register to be spilled/restored
```

Comment:
Should be "may cause".
I think you should have a single comment that talks about the heuristics for estimating register save/restore/spill costs. First, it is conservative to assume that *each* SIMD CSE will require an additional register to be saved/restored in prolog/epilog, as there may be disjoint CSE live ranges that can utilize the same register.
Second, this code presumes that the CSEs will be live arcoss a call, which may be true more often than not, but it also assumes that each use will require a restore, though it may be that multiple uses will not span a call.
While it may be fine to be conservative in these ways, I think we should accurately describe those conservative assumptions.

Reply:
OK